test/controllers/releases_controller_test.rb

```diff
@@ -2,28 +2,50 @@
 
 describe ReleasesController do
   let(:project) { projects(:test) }
+  let(:release) { releases(:test) }
 
-  describe "#create" do
-    let(:release_params) { { commit: "abcd" } }
+  as_a_viewer do
```

Comment:
structuring by viewer/deployer/admin give the test a nice flow/structure, since an action is only tested once and does not need to repeat the scopes multiple times, the alternative would be to do viewer/deployer etc inside a `describe action` block ...